### DISCUSSION TITLE
Which for Windows on non-GPU machine

### DISCUSSION THREAD

**Discussion by abdulbadii** [LeelaChessZero/lc0#2045] [Q&A] [answered]
Which LC0 release binary for Windows run on non-GPU machine, ie only most common PC?
And what do DNNL, OpenBlas, OpenCL mean or stand for in elaborated explanation?

**Comment by mooskagh** [comment, marked as answer]
There is a table [here](https://lczero.org/play/download/#windows) describing which version to use for which hardware.

DNNL, OpenBlas and OpenCL are different libraries for tensor arithmetics that run on different hardware:
* https://en.wikipedia.org/wiki/OpenCL
* https://oneapi-src.github.io/oneDNN/v1.3/index.html
* https://www.openblas.net
